File: src/main/java/com/wikia/tibia/objects/HuntingPlaceSkills.java
package com.wikia.tibia.objects;

import com.fasterxml.jackson.annotation.JsonBackReference;
import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor(access = AccessLevel.PRIVATE)
@AllArgsConstructor(access = AccessLevel.PRIVATE)
public class HuntingPlaceSkills {

    private String areaname;
    private String lvlknights;
    private String lvlpaladins;
    private String lvlmages;
    private String skknights;
    private String skpaladins;
    private String skmages;
    private String defknights;
    private String defpaladins;
    private String defmages;
    @JsonBackReference
    private HuntingPlace huntingPlace;
}

File: src/main/kotlin/com/wikia/tibia/objects/HuntingPlaceSkills.kt
package com.wikia.tibia.objects

import com.fasterxml.jackson.annotation.JsonBackReference

data class HuntingPlaceSkills(
        private val areaname: String,
        private val lvlknights: String,
        private val lvlpaladins: String,
        private val lvlmages: String,
        private val skknights: String,
        private val skpaladins: String,
        private val skmages: String,
        private val defknights: String,
        private val defpaladins: String,
        private val defmages: String,
        @JsonBackReference private val huntingPlace: HuntingPlace
)